### DISCUSSION TITLE
TumblThree 2.9.0 released!

### DISCUSSION THREAD

**Discussion by thomas694** [TumblThreeApp/TumblThree#376] [Announcements]
### TumblThree 2.9.0

- Changes
  - Update of some translations
  - Replacement of CefSharp by WebView2

- Features
  - Faster scans and saving bandwidth by not downloading the same information again

- Bug fixes
  - Occasional problem displaying error message
  - Filename contains converted HTML tags when using title token on newTumbl blogs
  - Error parsing some "Likes" posts (#374)

&nbsp;
*Note: From this version on the "[MS Edge Webview2 Runtime](https://learn.microsoft.com/en-us/microsoft-edge/webview2/)" is needed. If not yet installed, you need to [download](https://go.microsoft.com/fwlink/p/?LinkId=2124703) and install it, in order for TumblThree to work properly. On application start the requirement is checked and the download link offered.*

<hr /><em>This discussion was created from the release <a href='https://github.com/TumblThreeApp/TumblThree/releases/tag/v2.9.0'>TumblThree-v2.9.0</a>.</em>

**Comment by TomInVa57** [comment]
Hi. Getting error parsing post! on all blogs now :(

**Reply by TomInVa57** [reply]
Also, getting errors "A post from(blog name) was not parsable. Setting 'posts per page' in the Details to a lower value might increase the downloadable content" error. Haven't seen that on in a very long time. Reverting back to previous version :(

**Reply by thomas694** [reply]
> Hi. Getting error parsing post! on all blogs now :(

What types of blogs do you have? The normal blog types seem to work, only with LikedBy/Likes there's currently a problem, mainly due to the new option "download texts" which was added a few weeks ago. The many subtypes and combination possibilities aren't handled correctly all the time yet.
A workaround is to disable the option until the next version.
